src/examples/02_animations/custom_VMobject.py

```diff
@@ -0,0 +1,25 @@
+from manimlib.imports import *
+
+
+class MyDotGrid(VMobject):
+    def __init__(self):
+        VMobject.__init__(self)
+        dot1 = Dot(fill_color=GREEN).shift(LEFT)
```

Comment:
The colors are not defined (at least in manimlib 0.1.6, but manim-gallery depends on this version)

Reply:
Hrm - I just saw that in the other pull request, those colors are also used. PyCharm complains about them not being defined, but actually they are. Those damn `*` imports…

Reply:
I know, this is a little annoying..